Entity/TranslationRepository.php

```diff
@@ -14,7 +14,7 @@ class TranslationRepository extends EntityRepository
     /**
      * @return \DateTime|null
      */
-    public function getLatestTranslationUpdatedAt()
+    public function getLatestTranslationUpdatedAt(): \DateTime|null
```

Comment:
I would prefer `?DateTime` instead, with a `use DateTime` at the top of the file

Reply:
All updated. Also since PECL in mongo release 2.0.0, I just update for allow old version and new versions of mongo-php extension